Evaluation/Input/evaluation_plan.json

```diff
@@ -328,85 +311,82 @@
             "Project": "Alfentanil",
             "Name": "Alfentanil",
             "Type": "Compound",
-            "SectionId": 31
+            "SectionReference": "final-input-parameters"
         },
         {
             "Project": "Alfentanil",
             "Name": "Solution",
             "Type": "Formulation",
-            "SectionId": 31
+            "SectionReference": "final-input-parameters"
         }
     ],
     "Sections": [
         {
-            "Id": 1,
-            "Title": "1 Introduction",
+            "Reference": "introduction",
+            "Title": "Introduction",
```

Comment:
* Use `Reference `instead of `Id `in every section
* Remove chapter number from the title